app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TournamentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def saveTournamentResults
+    #byebug
+    
+    #TODO: adding validation to the models
+    tournament = Tournament.new
+
+    tournament.tournament_id = params["tournamentId"]
+    tournament.start_date = params[:startDate]
+    tournament.end_date = params[:endDateTime]
+    results = params[:results]
+    results.each do |i|
+      player = Player.new
+      player.user_id = i[:userId]
+      player.correct_questions = i[:correctQuestions]
+      player.incorrent_questions = i[:incorrectQuestions]
+      player.tournament_id = tournament.tournament_id
```

Comment:
This is very confusing - the Rails convention would be that player.tournament_id would hold tournament.id, not tournament.tournament_id
That's how ActiveRecord knows how to find it in the DB with the belongs_to relation

Reply:
no sure I got you.